lib/proxy.js

```diff
@@ -137,12 +137,13 @@ function buildResultingMiddleware (host, options, stream) {
     var reqOpts = {
       url     :  URL.format(proxyUrl),
       method  : req.method,
-      headers : _.defaults(options.headers || {}, req.headers),
+      // headers : _.defaults(options.headers || {}, req.headers),
+      headers : options.headers,
```

Comment:
I do agree with that and like the idea of it being as unobtrusive as possible. I guess i am just worried that is we are having issues with supertest headers who knows what other types of headers will cause issues against APIs.

So to be as unobtrusive as possible it may be nice to at least have the options to add/remove/modify headers sent to the API